webpack.config.js

```diff
@@ -71,14 +71,18 @@ module.exports = (env, argv) => {
                     test: /\.css$/,
                     use: [
                         styleLoaderImpl,
-                        { loader: 'css-loader', options: { minimize: true } }
+                        { loader: 'css-loader', options: { } }
+                        // https://github.com/webpack-contrib/css-loader/issues/863
+                        //                        { loader: 'css-loader', options: { minimize: true } }
```

Comment:
Should we just remove it instead of commenting out? It seems the `minimize` option has been removed from the API. If we want to minimize the CSS, we need to start using some other way, like:
https://github.com/webpack-contrib/mini-css-extract-plugin#minimizing-for-production